src/strategies/saddle-vesdl-voting-power/index.ts

```diff
@@ -0,0 +1,34 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller, getBlockNumber } from '../../utils';
+
+export const author = 'saddle-finance';
+export const version = '0.1.0';
+
+const abi = [
+  'function getPriorVotes(address account, uint256 block) external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag =
+    typeof snapshot === 'number' ? snapshot : await getBlockNumber(provider);
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) =>
+    multi.call(address, options.address, 'getPriorVotes', [address, blockTag])
+  );
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, balance]) => [
+      address,
+      parseFloat(formatUnits(balance, options.decimals))
+    ])
```

Comment:
Hey @nguyenaiden For this strategy, can you try the `contract-call` strategy and see if it works? Thanks :)

Reply:
@ChaituVR Could you explain this further? I'm not sure what you meant by trying the `contract-call` strategy.

Reply:
I am this strategy https://snapshot.org/#/strategy/contract-call , you can directly call a method from a contract, try playing with it in the playground here https://snapshot.org/#/playground/contract-call